app/javascript/components/get-weather-query.jsx

```diff
@@ -43,49 +43,56 @@ export const WeatherQuery = ({
   WeatherFrag,
   queryName,
   cityName,
-}) => (
-  <Query
-    query={getLocationWeather(WeatherFrag, queryName)}
-    variables={{ cityName }}
-  >
-    {({ loading, error, data, subscribeToMore }) => {
-      if (loading) {
-        return <LoadingMessage />;
-      }
-      if (error) {
+  resetTimer,
+}) => {
+  return (
```

Comment:
Do we need to switch from the implicit return to an explicit return? If so, lets make that a refactor that precedes the actual change